### PR TITLE
(MAINT) README updates for CI configs

### PR DESCRIPTION
**Pull request by scotje**
ping @glennsarti @rodjek 

We could probably stand to flesh out what "Officially Support" and "Community Supported" mean

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,6 +137,78 @@ Travis uses a .travis.yml file in the root of your repository to learn about you
 |coverage_report|Enable [rspec-puppet coverage reports](https://rspec-puppet.com/documentation/coverage/). Defaults to `false`|
 |minimum_code_coverage_percentage|The desired code coverage percentage required for tests to pass. Defaults to `0`|
 
+## Configurations for Continuous Integration (CI) Environments
+
+The following keys allow you to configure and customize various third-party continuous integration (CI) environments for automatically testing your module.
+
+### Officially Supported CI Configurations
```
Comment (glennsarti) [inline README.md]: IMO .... this doesn't really specify what "supported" means.

Comment (scotje) [reply README.md]: Yeah, I mostly wanted to get this started, we can workshop how the two categories are actually described.

**Comment by glennsarti** [comment]
@scotje Ping. Open since Sept.

**Comment by pmcmaw** [comment]
Hey @scotje :-)

Just having a large tool triage and wondering if this is something you are still working on?

**Comment by DavidS** [comment]
I've extracted the typo fixes to #357 and will close the rest of this PR until @binford2k can clarify the required wording around support.

**Pull request closed without merging**